client/state/selectors/get-current-route-generic.js

```diff
@@ -0,0 +1,18 @@
+import { getSiteFragment } from 'calypso/lib/route';
+import getCurrentRoute from './get-current-route';
+
+/**
+ * Gets the last route set by a ROUTE_SET actionand replaces the siteFragment with ":site"
+ * @param {Object} state - global redux state
+ * @returns {string} current route value
+ */
+export const getCurrentRouteGeneric = ( state ) => {
```

Comment:
I like `getCurrentRoutePattern `  here is the change with tests https://github.com/Automattic/wp-calypso/pull/85102/commits/6b7deccec13dbe34cbc52d58aaf13098e5db4947
Thanks to you name suggestion I found `getCurrentRouteParameterized` already exists, but it doesn't work for our case because it returns `null` if there is no selected `siteId`.